cluster/src/test/java/com/hartwig/pipeline/execution/vm/GoogleComputeEngineTest.java

```diff
@@ -206,9 +207,9 @@ public void usesPublicNetworkIfNoPrivateSpecified() {
     }
 
     @Test
-    public void usesPrivateNetworkWhenSpecified() {
+    public void usesNetworkAndSubnetWhenSpecified() {
```

Comment:
Should we also have a test to ensure it defaults to the network when the subnet isn't given?